### DISCUSSION TITLE
project recipe

### DISCUSSION THREAD

**Discussion by MarciolimaDev** [community/community#111031] [Other Feature Feedback, Questions, & Ideas]
### Select Topic Area

General

### Body

https://github.com/MarciolimaDev/projectRecipes

**Comment by IvankaKHan** [comment]
Hi

**Comment by MarciolimaDev** [comment]
Hi, how's it going?

Em qua., 13 de nov. de 2024 03:19, Ivanka ***@***.***>
escreveu:
